### PR TITLE
jmeter base class

### PR DESCRIPTION
**Pull request by bserdar**

### CODE REVIEW

File: jmeter-tests/src/main/java/com/redhat/lightblue/jmeter/AbstractTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        LightblueHttpClient httpClient;
+        configFile=context.getParameter("ConfigFile",PropertiesLightblueClientConfiguration.DEFAULT_CONFIG_FILE);
+        getLogger().debug("Getting client, config="+configFile);
+        if (configFile != null) {
+            lbClient = new LightblueHttpClient(configFile);
+        } else {
+            lbClient = new LightblueHttpClient();
+        }
+        getLogger().debug("lbClient="+lbClient);
+        name = context.getParameter(TestElement.NAME);
```
Comment (dcrissman) [inline jmeter-tests/src/main/java/com/redhat/lightblue/jmeter/AbstractTest.java]: What about using the LightblueExternalResource junit rule to stand a lightblue instance?

Comment (bserdar) [inline jmeter-tests/src/main/java/com/redhat/lightblue/jmeter/AbstractTest.java]: The point of this is to call a remote lightblue cluster.

**Comment by dcrissman** [comment]
@bserdar looks like travis is failing because you are relying on an older version of the lightblue client. Would you please bump that to the most recent?

**Comment by bserdar** [comment]
Done

File: jmeter-tests/src/main/java/com/redhat/lightblue/jmeter/AbstractTest.java (hunk 1 above)
Comment (dcrissman) [inline jmeter-tests/src/main/java/com/redhat/lightblue/jmeter/AbstractTest.java]: Ok, so the intention is that implementing test classes would provide their own lightblue cluster.

Comment (bserdar) [inline jmeter-tests/src/main/java/com/redhat/lightblue/jmeter/AbstractTest.java]: Yes, you have to have a running lightblue somewhere to use these jmeter
tests.

**Pull request merged**